polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -671,6 +678,7 @@ pub mod pallet {
 		#[codec(index = 23)]
 		TooManyReserves,
 		/// Local XCM execution incomplete.
+		/// @deprecated Use `LocalExecutionIncompleteWithError` for more detailed error information
```

Comment:
What is `@deprecated`? :)

Do we want to really deprecate it with:
```
#[deprecated(
        since = "1.2.3",
        note = "Use `NewVariant` instead"
    )]
``` 
or just left here some TODO/issue for later removal?